src/earn/poolInfo.ts

```diff
@@ -32,3 +36,47 @@ export async function fetchAavePoolInfo(assetAddress: Address) {
     throw err
   }
 }
+
+/** @beta - Exclude from Knip dep check */
+export async function fetchAavePoolUserBalance({
+  assetAddress,
+  walletAddress,
+}: {
+  assetAddress: Address
+  walletAddress: Address
+}) {
+  try {
+    if (!assetAddress) {
+      throw new Error('No asset address provided')
+    }
+    if (!walletAddress) {
+      throw new Error('No wallet address provided')
+    }
+    Logger.debug(TAG, 'Fetching Aave pool user balance', { assetAddress, walletAddress })
+    const result = await publicClient[Network.Arbitrum].readContract({
+      abi: AavePool,
+      address: networkConfig.arbAavePoolV3ContractAddress,
+      functionName: 'getReserveData',
+      args: [assetAddress],
+    })
+
+    const { aTokenAddress } = result as { aTokenAddress: `0x${string}` }
```

Comment:
use `Address` instead of `0x${string}`?

Reply:
Fixed in 5bc1b87684d4f940a27421f3842738d9ec374230!